comid/ccareferencevalue.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"fmt"
+)
+
+// CCARefValID stores a CCA refval-id with CBOR and JSON serializations
+type CCARefValID struct {
```

Comment:
I think this should be renamed `CCAPlatformConfigID` (and the file names accordingly).

Also: the struct wrapper is superfluous as it's only got one field.

Unless we want to make the label optional to allow anonymous/unlabelled config values.